### ISSUE TITLE
Using pom.xml as default value for settings_file

### ISSUE THREAD

**Issue by radek-svanda-elder** [CircleCI-Public/maven-orb#3]
**Orb Version**
1.0.0

**Describe the bug**

There is this piece in files [commands/with_cache.yml](https://github.com/CircleCI-Public/maven-orb/blob/master/src/commands/with_cache.yml) 

```yml
  settings_file:
        description: Specify a custom settings file to use (optional)
        type: string
        default: pom.xml
```

The default value will pass this condition later in the file

```yml
  - when:
      condition: << parameters.settings_file >>
```

Which leads in maven warning in _Install dependencies_ step

```
#!/bin/bash -eo pipefail
./mvnw dependency:go-offline --settings 'pom.xml'

[WARNING] 
[WARNING] Some problems were encountered while building the effective settings
[WARNING] Expected root element 'settings' but found 'project' (position: START_TAG seen ...pache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">... @3:110)  @ /tmp/workspace/pom.xml, line 3, column 110
[WARNING] 
```

**To Reproduce**

```yml
  maven-orb-test:
    executor:
      name: maven/default
      tag: '11.0.6'
    steps:
      - checkout
      - maven/with_cache:
          steps:
            - run: mvn compile
```

**Expected behavior**

Probably remove the default value since `pom.xml` is not valid `settings.xml` file.

It is not a high priority bug but current situation is not valid.

**Additional context**

Same applies for [jobs/test.ym](https://github.com/CircleCI-Public/maven-orb/blob/master/src/jobs/test.yml) file.

**Comment by ZachSand** [comment]
Just in case this wasn't obvious for anyone seeing this, a workaround until this is fixed is to just make the settings_file blank in your config.yml
```
paramters:
  settings_file:
```

**Comment by lokst** [comment]
@radek-svanda-elder Thanks for reporting this, we'll look into a more suitable default value

**Comment by lokst** [comment]
@radek-svanda-elder This is now resolved in version `1.0.1` of the orb
